composer.json

```diff
@@ -10,7 +10,8 @@
   ],
   "require": {
     "php": ">=7.1",
-    "fond-of-spryker/brand": "*"
+    "spryker/glue-application": "^1.0.0",
```

Comment:
Checked for possible versions

Reply:
Why did you remove "fond-of-spryker/brand"? There is a client bridge to brand in glue part of this module.

Reply:
I just changed the version to "dev-master". Therefore Git show a remove line and an add line.